System/Process.hs

```diff
@@ -200,6 +200,13 @@ always the desired behavior. In cases where you would like to leave the
 instead. All created @Handle@s are initially in text mode; if you need them
 to be in binary mode then use 'hSetBinaryMode'.
 
+@/ph/@ contains a handle to the running process.  On Windows
+'use_process_jobs' can be set in CreateProcess in order to create an
```

Comment:
you removed the wrong one :) This should be

```
@/ph/@ contains a handle to the running process.  On Windows
'use_process_jobs' can be set in CreateProcess in order to create an
I/O completion port to monitor a process tree's progress.  If it is
set then that port is also returned inside @/ph/@.  The port can be
used to kill all running subprocesses.  This feature has been
available since 1.5.0.0.
```

We simply no longer use completion ports to monitor for processes.
We only use a job object. See `waitForJobCompletion`. So this should be
something like

```
@/ph/@ contains a handle to the running process.  On Windows
'use_process_jobs' can be set in CreateProcess in order to create a
Win32 Job object to monitor a process tree's progress.  If it is
set then that job is also returned inside @/ph/@.  The job and process
can be used to kill all running sub-processes.  This feature
has been available since 1.5.0.0.
```

Though realistically speaking this is an implementation detail. The user is expected to
only use `ph`, e.g. `terminateProcess` takes `ph` not the job.

I'm not sure how much of this we should mention or should stay internal @snoyberg ?

Reply:
Fixed accordingly.

I sympathise with the wish to not say too much about implementation details but as things stand the documentation is simply wrong. Could we at least merge this PR and then think about how much of the internals are mentioned in a separate PR?